example/satosa/openid4vci_frontend.yaml

```diff
@@ -0,0 +1,677 @@
+                - org.iso.18013.5.1
+                - expiry_date
+              display:
+                - name: Data di scadenza (YYYY-MM-GG)
+                  locale: it-IT
+                - name: Expiry Date (YYYY-MM-GG)
+                  locale: en-US
+            - path:
+                - org.iso.18013.5.1
+                - issuing_country
+              display:
+                - name: Paese di rilascio
+                  locale: it-IT
+                - name: Issuing Country
+                  locale: en-US
+            - path:
+                - org.iso.18013.5.1
+                - issuing_authority
+              display:
+                - name: Autorità di rilascio
+                  locale: it-IT
+                - name: Issuing Authority
+                  locale: en-US
+            - path:
+                - org.iso.18013.5.1
+                - document_number
+              display:
+                - name: Numero di documento
+                  locale: it-IT
+                - name: Document Number
+                  locale: en-US
+            - path:
+                - org.iso.18013.5.1
+                - portrait
+              display:
+                - name: Foto codificata in base64
+                  locale: it-IT
+                - name: Portrait base64 encoded
+                  locale: en-US
+            - path:
+                - org.iso.18013.5.1
+                - driving_privileges
+              display:
+                - name: Elenco delle categorie di abilitazione e relativi dettagli su restrizioni/condizioni
+                  locale: it-IT
+                - name: Driving Privileges and related restrictions/conditions details
+                  locale: en-US
+            - path:
+                - org.iso.18013.5.1
+                - un_distinguishing_sign
+              display:
+                - name: Codice identificativo della Nazione
+                  locale: it-IT
+                - name: Distinguishing sign of the issuing country
+                  locale: en-US
+      jwks: *metadata_jwks
+      trust_frameworks_supported:
+        - it_cie
+        - it_wallet
+        - eudi_wallet
+      evidence_supported:
+        - vouch
+    federation_entity: &federation_entity_metadata
+      organization_name: Organization Name
+      homepage_uri: https://eaa-provider.example.org/
+      policy_uri: https://eaa-provider.example.org/public/privacy_policy.html
+      tos_uri: https://eaa-provider.example.org/public/info_policy.html
+      logo_uri: https://eaa-provider.example.org/public/logo.svg
+      contacts:
+        - informazioni@example.it
+        - protocollo@pec.example.it
+
+  # this is the custom configuration for credential management.
+  credential_configurations:
+    lookup_source: openid4vci #this key is the same one found in `internal_attributes.yaml`, used for lookup filter for attributes
+    entity_configuration_exp: 800
+    entity_default_sig_alg: *default_sig_alg
```

Comment:
entity default sig alg is a completely wrong name

the subject is the credential and not the entity (which would belong tot the openid federation configuration)

default sig algs msut be defined per credential type, theresore sd-jet and mdoc cbor may have different default sig alg